### PR TITLE
[T4A6][W11-B3] Fan Weiguang

### PR DESCRIPTION
**Pull request by fanwgwg**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package seedu.addressbook.common;
+
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/common/UtilsTest.java]: Don't forget to put header comment.

File: src/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import static org.junit.Assert.*;
+import java.util.ArrayList;
+import org.junit.Test;
+
+public class UtilsTest {
+	
+	@Test
+	public void isAnyNull_noArgs() {
+		boolean result = Utils.isAnyNull();
+		assertEquals(false, result);
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/common/UtilsTest.java]: You can also use `assertFalse` and put the argument to tell what is the problem, like #1199.

**Comment by jeffryhartanto** [comment]
Some comments added. Please close the PR after reading comments. Good job.

**Comment by fanwgwg** [comment]
Comments reviewed. Thank you very much!

**Pull request closed without merging**